worker/src/utils.rs

```diff
@@ -1,2 +1,19 @@
 extern crate console_error_panic_hook;
+use anyhow::{anyhow, Result};
+
 pub use self::console_error_panic_hook::set_once as set_panic_hook;
+
+pub fn extract_signature(text: String) -> Result<(String, String)> {
```

Comment:
I suppose you can make the Twitter function use it.

Reply:
Yeah, I feel like this function is going to be used a lot.